.github/workflows/build.yaml

```diff
@@ -0,0 +1,25 @@
+name: build
+on: [push]
```

Comment:
If there ever were some external PRs, I would suggest building on those, too.
But I guess this is enough now.